app/src/main/java/rick_and_morty/ui/characters/composables/CharacterList.kt

```diff
@@ -0,0 +1,84 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import rick_and_morty.ui.characters.CharactersViewModel
+
+@Composable
+fun CharacterList() {
+    val charactersViewModel = viewModel(modelClass = CharactersViewModel::class.java)
+
+    var page = 1;
+
+    charactersViewModel.getCharacters(page)
```

Comment:
this should probably be part of `LaunchedEffect(Unit)`.... in this scenario this would never recompose, but if it does it would be a problem.

also this is skipping the `initialLoad` check... solved by moving the check inside the VM

Reply:
Done